backend/function-apps/dataflows/dataflows.ts

```diff
@@ -1,24 +1,83 @@
 import * as dotenv from 'dotenv';
 import { initializeApplicationInsights } from '../azure/app-insights';
 
-import { setupMigrateCases } from './import/migrate-cases';
-import { setupSyncCases } from './import/sync-cases';
-import { setupSyncOrders } from './import/sync-orders';
-import { setupSyncOfficeStaff } from './import/sync-office-staff';
-import { setupMigrateConsolidations } from './import/migrate-consolidations';
+import SyncCases from './import/sync-cases';
+import SyncOrders from './import/sync-orders';
+import SyncOfficeStaff from './import/sync-office-staff';
+import MigrateCases from './import/migrate-cases';
+import MigrateConsolidations from './import/migrate-consolidations';
+
+const MODULE_NAME = 'DATAFLOWS_SETUP';
+
+type DataflowSetup = {
+  MODULE_NAME: string;
+  setup: () => void;
+};
+
+class DataflowSetupMap extends Map<string, () => void> {
+  add(...dataflows: DataflowSetup[]) {
+    for (const dataflow of dataflows) {
+      this.set(dataflow.MODULE_NAME, dataflow.setup);
+    }
+  }
+
+  list() {
+    return [...this.keys()];
+  }
+
+  enable(...dataflowNames: string[]) {
+    const cleanNames = dataflowNames.map((name) => name.trim().toUpperCase());
+    const status = [];
+
+    for (const dataflowName of cleanNames) {
+      if (this.has(dataflowName)) {
+        this.get(dataflowName)();
+        status.push([dataflowName, true]);
+      } else {
+        console.warn(MODULE_NAME, 'Dataflow name', dataflowName, 'not found');
+      }
+    }
+    for (const dataflowName of this.list()) {
+      if (!cleanNames.includes(dataflowName)) {
+        status.push([dataflowName, false]);
+      }
+    }
+    return status;
+  }
+}
+const dataflows = new DataflowSetupMap();
 
 // Setup environment and AppInsights.
 dotenv.config();
 initializeApplicationInsights();
 
-// Setup the recurring synchronization.
-setupSyncCases();
-setupSyncOfficeStaff();
-setupSyncOrders();
+// Register data flows.
+dataflows.add(SyncCases, SyncOfficeStaff, SyncOrders, MigrateCases, MigrateConsolidations);
+
+// Log the list of registered data flows.
+const registeredDataflows = dataflows.list().join(', ');
+console.log(MODULE_NAME, 'Registered Dataflows', registeredDataflows);
+
+// Enable the data flows specified in from the configuration env var.
+const envVar = process.env.CAMS_ENABLE_DATAFLOWS;
+const dataflowNames = envVar.split(',');
+const status = dataflows.enable(...dataflowNames);
+
+// Log the status of each registered data flow.
+status.forEach((s) => {
+  console.log(MODULE_NAME, 'Enabled', s);
```

Comment:
If the `enable` function is renamed to `setup` then change what is logged.
```suggestion
  console.log(MODULE_NAME, s);
```